## Hypothesis Multivariate Hypothesis

In [ ]:
# Imports
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt



dataset_path = "../data/merged_dataset.csv"

df = pd.read_csv(dataset_path)

#df.head()


filtered_df = df[(df['year'].isin([2000, 2005])) | (df['year'].isin([2010, 2015]))]
filtered_df.head()

,country_name,country_code,year,suicide_data,homicide_data,intentional_death,gini_index
1,albania,alb,2005,6.91,154.0,160.91,0.31
5,albania,alb,2015,3.58,64.0,67.58,0.33
15,armenia,arm,2005,3.86,58.0,61.86,0.36
20,armenia,arm,2010,5.97,56.0,61.97,0.30
25,armenia,arm,2015,4.58,75.0,79.58,0.32


### Hypotheses

- H₀: The Gini index does not predict suicide or homicide rates.
This means that economic inequality has no statistically significant effect on intentional death outcomes (suicide and homicide). Any observed differences in suicide or homicide rates are assumed to be due to random variation rather than inequality.
- H₁: The Gini index significantly predicts suicide or homicide rates.
This means that economic inequality has a statistically significant effect on intentional deaths. As inequality increases or decreases, suicide and homicide rates are expected to change in a systematic way.


##### Rationale — Why is this hypothesis relevant?

This hypothesis is directly relevant to your research question—“How does a country’s economic inequality (Gini index) affect intentional death rates?”—because:

- Economic inequality is a key structural factor that influences social stress, access to resources, and social cohesion.

- Suicide and homicide are known to be affected by economic, psychological, and social pressures, all of which are linked to inequality.

- Countries with higher inequality often experience greater psychosocial stress, social fragmentation, and reduced access to support systems, which can elevate the rates of intentional deaths.

- Testing this hypothesis allows you to determine whether inequality is a significant predictor, statistically and substantively, using large-scale data.

- If the Gini index is significant (as your regression results show with p < 0.001), it suggests that policy efforts targeting socioeconomic inequality may help reduce suicide and homicide rates.

The hypothesis is important because it tests whether inequality—an underlying socioeconomic factor—plays a measurable role in shaping rates of intentional harm. This aligns directly with your goal of understanding how inequality influences national-level health and safety outcomes.

### Variables

Variables in the Hypothesis
1. Variables Involved

Your hypothesis examines the relationship between:
- Gini index (economic inequality)
- Suicide rate
- Homicide rate

Often combined as intentional death rate (depending on your model).

2. Variable Types
• Gini Index
- Type: Continuous numerical variable
- Description: Ranges from 0 to 1 in your dataset (0 = perfect equality, 1 = extreme inequality).

• Suicide Rate
- Type: Continuous numerical variable
- Description: The number of suicides per 100,000 population (or similar standardized measure).

• Homicide Rate
- Type: Continuous numerical variable
- Description: The number of homicides per 100,000 population.

• Intentional Death Rate (if used)
- Type: Continuous numerical variable
- Description: Combined suicide + homicide values.

3. Independent vs. Dependent Variables
Independent Variable (Predictor):
- Gini Index
This is the variable whose effect you are testing. It represents economic inequality.

Dependent Variables (Outcomes):
- Suicide Rate
- Homicide Rate
(or Intentional Death Rate, if combined)

These are the outcomes you expect to change as inequality changes.

### MANOVA 

In [5]:
from statsmodels.multivariate.manova import MANOVA

# MANOVA: suicide & homicide predicted by gini
maov = MANOVA.from_formula("suicide_data + homicide_data ~ gini_index", data=filtered_df)
print(maov.mv_test())

                  Multivariate linear model
                                                             
-------------------------------------------------------------
       Intercept        Value  Num DF  Den DF  F Value Pr > F
-------------------------------------------------------------
          Wilks' lambda 0.6330 2.0000 217.0000 62.9191 0.0000
         Pillai's trace 0.3670 2.0000 217.0000 62.9191 0.0000
 Hotelling-Lawley trace 0.5799 2.0000 217.0000 62.9191 0.0000
    Roy's greatest root 0.5799 2.0000 217.0000 62.9191 0.0000
-------------------------------------------------------------
                                                             
-------------------------------------------------------------
       gini_index       Value  Num DF  Den DF  F Value Pr > F
-------------------------------------------------------------
          Wilks' lambda 0.7565 2.0000 217.0000 34.9173 0.0000
         Pillai's trace 0.2435 2.0000 217.0000 34.9173 0.0000
 Hotelling-Lawley trace 0.

p < 0.001

This means the Gini index significantly predicts the combined outcomes (suicide + homicide).

1. Wilks’ Lambda = 0.7565 (p < .001)

Wilks’ Lambda measures how much variance in the dependent variables is not explained by the predictor.
- Values closer to 0 → strong effect
- Values closer to 1 → weak effect
- Your value: 0.7565 → moderate effect
- p < .001 → statistically significant

👉 Interpretation:
Gini index explains a significant portion of the variance in suicide & homicide rates.

2. Pillai’s Trace = 0.2435 (p < .001)

This measures the proportion of explained variance:

👉 24.35% of the variance in the combined outcomes (suicide + homicide) is explained by inequality.

This matches your earlier regression that showed ~15–16% but expands to the multivariate case.

3. Hotelling-Lawley & Roy’s Root

These also confirm the same result:

- Strong effect
- Highly significant
- Consistent across all multivariate tests

### Final Conclusion:

Economic inequality (Gini index) has a statistically significant multivariate effect on intentional death rates (suicide & homicide) across 220 observations. Higher inequality is associated with higher levels of suicide and homicide when examined together.